handler.go

```diff
@@ -269,7 +310,11 @@ var (
 	{{- if .IsDev }}
 		{{ .PluginReactPreamble }}
 		<script type="module" src="{{ .ViteURL }}/@vite/client"></script>
-		<script type="module" src="{{ .ViteURL }}/src/main.tsx"></script>
+		{{ if ne .ViteEntry "" }}
+			<script type="module" src="{{ .ViteURL }}/{{ .ViteEntry }}"></script>
+		{{ else }}
+			<script type="module" src="{{ .ViteURL }}/src/main.tsx"></script>
+		{{ end }}
```

Comment:
... and finally:

```suggestion
		{{- end }}
```

... as that would make the output a bit cleaner?

Reply:
Will do.